WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -46,6 +55,13 @@ class PostListMainViewModel @Inject constructor(
     private val _isFabVisible = MutableLiveData<Boolean>()
     val isFabVisible: LiveData<Boolean> = _isFabVisible
 
+    private val _filterAuthorListItems = MutableLiveData<List<AuthorFilterListItemUIState>>()
+    val filterAuthorListItems: LiveData<List<AuthorFilterListItemUIState>> = _filterAuthorListItems
+
+    val filterOnlyUser: LiveData<Boolean> = _filterAuthorListItems.map { items ->
+        return@map items.firstOrNull { state -> state.isSelected } is AuthorFilterListItemUIState.Me
```

Comment:
Nitpick: `return@map` is not really necessary here. I think `return@xyz` usages should be utilized only when it makes things clearer which I don't think is the case here.

Reply:
Agreed, that was a hold over from when I was fine tuning how that map would work.